packages/mock-app/report.json

```diff
@@ -0,0 +1 @@
alInfo.tsx":{"path":"/Users/yenle/Documents/source/mock-app-new/tests-untp/packages/mock-app/src/components/CredentialInfo/CredentialInfo.tsx","statementMap":{"0":{"start":{"line":1,"column":0},"end":{"line":1,"column":39}},"1":{"start":{"line":2,"column":0},"end":{"line":2,"column":28}},"2":{"start":{"line":3,"column":0},"end":{"line":3,"column":61}},"3":{"start":{"line":6,"column":23},"end":{"line":42,"column":1}},"4":{"start":{"line":7,"column":25},"end":{"line":19,"column":23}},"5":{"start":{"line":8,"column":4},"end":{"line":10,"column":null}},"6":{"start":{"line":9,"column":6},"end":{"line":9,"column":29}},"7":{"start":{"line":12,"column":18},"end":{"line":12,"column":46}},"8":{"start":{"line":13,"column":17},"end":{"line":13,"column":71}},"9":{"start":{"line":13,"column":39},"end":{"line":13,"column":70}},"10":{"start":{"line":14,"column":4},"end":{"line":16,"column":null}},"11":{"start":{"line":15,"column":6},"end":{"line":15,"column":18}},"12":{"start":{"line":18,"column":4},"end":{"line":18,"column":34}},"13":{"start":{"line":22,"column":4},"end":{"line":25,"column":null}},"14":{"start":{"line":24,"column":6},"end":{"line":24,"column":23}},"15":{"start":{"line":26,"column":4},"end":{"line":26,"column":18}},"16":{"start":{"line":29,"column":2},"end":{"line":41,"column":4}},"17":{"start":{"line":44,"column":0},"end":{"line":44,"column":30}}},"fnMap":{"0":{"name":"(anonymous_1)","decl":{"start":{"line":6,"column":23},"end":{"line":6,"column":24}},"loc":{"start":{"line":6,"column":80},"end":{"line":42,"column":1}}},"1":{"name":"(anonymous_2)","decl":{"start":{"line":7,"column":33},"end":{"line":7,"column":36}},"loc":{"start":{"line":7,"column":38},"end":{"line":19,"column":3}}},"2":{"name":"(anonymous_3)","decl":{"start":{"line":13,"column":29},"end":{"line":13,"column":30}},"loc":{"start":{"line":13,"column":39},"end":{"line":13,"column":70}}},"3":{"name":"processIssuer","decl":{"start":{"line":21,"column":11},"end":{"line":21,"column":24}},"loc":{"start":{"line":21,"column":43},"end":{"line":27,"column":3}}}},"branchMap":{"0":{"loc":{"start":{"line":8,"column":4},"end":{"line":10,"column":null}},"type":"if","locations":[{"start":{"line":8,"column":4},"end":{"line":10,"column":null}}]},"1":{"loc":{"start":{"line":14,"column":4},"end":{"line":16,"column":null}},"type":"if","locations":[{"start":{"line":14,"column":4},"end":{"line":16,"column":null}}]},"2":{"loc":{"start":{"line":22,"column":4},"end":{"line":25,"column":null}},"type":"if","locations":[{"start":{"line":22,"column":4},"end":{"line":25,"column":null}}]},"3":{"loc":{"start":{"line":22,"column":8},"end":{"line":22,"column":52}},"type":"binary-expr","locations":[{"start":{"line":22,"column":8},"end":{"line":22,"column":34}},{"start":{"line":22,"column":38},"end":{"line":22,"column":52}}]}},"s":{"0":1,"1":1,"2":1,"3":1,"4":4,"5":4,"6":1,"7":3,"8":3,"9":4,"10":3,"11":1,"12":2,"13":4,"14":3,"15":1,"16":4,"17":1},"f":{"0":4,"1":4,"2":4,"3":4},"b":{"0":[1],"1":[1],"2":[3],"3":[4,3]}}}}
```

Comment:
I don’t think we should push the unit test results to the repository. Should we add this file to the `.gitignore` to prevent unnecessary code from being pushed?